android/app/src/main/AndroidManifest.xml

```diff
@@ -27,9 +26,8 @@
         <action android:name="android.intent.action.VIEW"/>
         <category android:name="android.intent.category.DEFAULT"/>
         <category android:name="android.intent.category.BROWSABLE"/>
-        <data android:scheme="com.shapeshift.droid_shapeshift"/>
+        <data android:scheme="shapeshift"/>
       </intent-filter>
     </activity>
-    <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" android:exported="false"/>
```

Comment:
https://docs.expo.dev/bare/upgrade/?fromSdk=51&toSdk=52

Check the upgrade helpers from 51 to 52 and 52 to 53 for references